cmd/kwil-admin/cmds/setup/genesis.go

```diff
@@ -0,0 +1,219 @@
+package setup
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/kwilteam/kwil-db/cmd/common/display"
+	"github.com/kwilteam/kwil-db/common/chain"
+	"github.com/spf13/cobra"
+)
+
+var (
+	genesisLong = `` + "`" + `genesis` + "`" + `creates a new genesis.json file.
+
+This command allows users to create genesis files without having to write them manually.
```

Comment:
Maybe without the third party verbage: "This command creates a new genesis file with optionally specified modifications."